presto-docs/src/main/sphinx/release/release-326.rst

```diff
@@ -0,0 +1,46 @@
+=========================
+Release 326 (25 Nov 2019)
+=========================
+
+General Changes
+---------------
+
+* Fix performance regression in queries involving ``JOIN``. (:issue:`2047`)
+* Fix accounting of semantic analysis time when queries are cancelled while queued. (:issue:`2055`)
+* Add :doc:`/connector/memsql`. (:issue:`1906`)
+* Improve performance of ``INSERT`` and ``CREATE TABLE ... AS`` queries containing redundant
+  ``ORDER BY`` clauses. (:issue:`2044`)
+* Improve performance when processing columns of ``MAP`` type. (:issue:`2015`)
+
+Security Changes
+----------------
+
+* Deprecate Kerberos in favor of JWT for :doc:`/security/internal-communication`. (:issue:`2032`)
+
+Hive Changes
+------------
+
+* Fix table creation error for tables with S3 location when using ``file`` metastore. (:issue:`1664`)
+* Fix a compatibility issue with the CDH 5.x metastore which results in stats
+  not being recorded for :doc:`/sql/analyze`. (:issue:`973`)
+* Improve performance for Glue metastore by fetching partitions in parallel. (:issue:`1465`)
+* Improve performance of ``sql-standard`` security. (:issue:`1922`, :issue:`1929`)
+
+RPM Changes
+-----------
+
+* Allow running Presto with Java versions newer than 11. (:issue:`2057`)
+
+Phoenix Connector Changes
+-------------------------
+
+* Collect statistics on the count and duration of each call to Phoenix. (:issue:`2024`)
+
```

Comment:
I am not sure if we should mention this, but it will affect all custom jdbc-based connector implementations.

```
Base JDBC Connector changes (aka second level of SPI)
---
 - Require to provide ``JdbcClient`` with ``ConnectionFactory`` annotated with ``@ForBaseJdbc``. (:issue:`2102`).
```

Reply:
I don't think it's necessary to call this out